sdk/core/azure-core/CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+# Release History
+
+## 1.0.0-beta.3 (Unreleased)
+
+## 1.0.0-beta.2 (2020-10-05)
+
+### New Features
+
+- Added pagination support.
```

Comment:
We can be a little more detailed in the changelog. Usually where we want to be succint is in the release notes. By the way, is @samvaity aware this is releasing this cycle along with the Chat SDK?

Reply:
I don't know if we have someone coordinating publishing the release notes or all other related stuff for mobile. //cc @czubair.

Reply:
@bsiegel I believe the docs are still like last time for mobile, i.e. it goes to https://azure.github.io/azure-sdk-for-android/. Please confirm

Reply:
And mobile android/iOS is yet to be onboarded to https://azure.github.io/azure-sdk/releases/latest/index.html. This means, there is no change in process since the last beta.1 release.

Reply:
Confirmed the doc publishing to github.io is still manual

Reply:
To close on this:

1. JavaDoc will go to GitHub [pages](https://azure.github.io/azure-sdk-for-android/).
2. The onboarding of mobile to the wider Azure SDK release [page](https://azure.github.io/azure-sdk/releases/latest/index.html) can be a follow-up PR to the [here](https://github.com/Azure/azure-sdk/tree/master/releases/2020-10).

Reply:
updated the changelog entry.